codebuild_specs/build_windows.yml

```diff
@@ -4,10 +4,6 @@ env:
 phases:
   build:
     commands:
-      - cd scripts
-      - yarn
```

Comment:
looks like build is failing and is expecting the scripts directory to have yarn install run there
<img width="1259" alt="image" src="https://github.com/user-attachments/assets/6329bc25-afd3-40ac-a421-eb32a37ca9b1">
It produces the scripts cache

Reply:
ok, does that mean these lines should be re-instated?

Reply:
yes for lines 7 & 8, here, and in the other file where we also do cd scripts/yarn

Reply:
Should I also add cd .. after yarn for this script?

Reply:
updated build_windows.yml and action.yml to reinstate code that runs yarn install in the scripts file

Reply:
it has been added